src/scss/use-cases/_article.scss

```diff
@@ -1,42 +1,32 @@
-/// Body text styles
-/// @brand master
-@mixin oTypographyBody {
-	@include oTypographySerif($scale: 1, $line-height: 28px);
-	@include oTypographyMargin($top: 0, $bottom: 7);
-	@include oColorsFor('o-typography-body');
-}
-
 /// Style for timestamps
 /// @brand master
 @mixin oTypographyTimestamp {
   @include oTypographySans($scale: 0);
-  @include oColorsFor(o-typography-timestamp);
   @include oTypographyMargin($bottom: 4);
   display: inline-block;
   text-transform: uppercase;
+  color: oColorsGetPaletteColor('black-60');
 }
 
 /// Article Standfirst styles
 /// @brand master
 @mixin oTypographyStandfirst {
 	@include oTypographySans($scale: 2);
 	@include oTypographyMargin($top: 0, $bottom: 9);
-	@include oColorsFor('o-typography-standfirst');
+	color: oColorsGetPaletteColor('black-60');
```

Comment:
does this not fall under branding because we plan on creating `o-article-stuff`?

Reply:
Sorry I'm not sure what your question is? `oTypographyStandfirst` will hopefully be taken out of `o-typography` entirely in the next breaking.

Reply:
I mean, we have a `_brand.scss` page now, but are setting this color explicitly in this mixin—why doesn't it get a '_oTypographyGet(...)` to fetch the a brand colour?

Reply:
I did it this way as it doesn't need to be configurable by other brands.